src/NuGet.Services.AzureSearch/Models/SearchDocument.cs

```diff
@@ -92,6 +95,17 @@ public class UpdateDownloadCount : UpdatedDocument, IDownloadCount
             public double? DownloadScore { get; set; }
         }
 
+        /// <summary>
+        /// Used when updating just the fields related to the default search exclusion of a document. Note that this model does
+        /// not need any analyzer or other Azure Search attributes since it is not used for index creation. The
+        /// <see cref="Full"/> and its parent classes handle this.
+        /// </summary>
+        [SerializePropertyNamesAsCamelCase]
+        public class UpdateIsExcludedByDefault : UpdatedDocument, IIsExcludedByDefault
```

Comment:
I don't think we need this document right? `Full` is enough? I don't think you even need the new interface

Reply:
I had this for future integration with auxiliary2azuresearch, will get rid of it. I think the interface is useful for explicitly defining the `PopulateIsExcludedByDefault` method. We could merge this with DownloadCount updates, but would avoid doing that until we are ready to add this updation logic aux2azs.